cumulusci/tasks/salesforce/promote_package_version.py

```diff
@@ -79,6 +86,30 @@ def _run_task(self) -> None:
         target_package_info = self._get_target_package_info(version_id)
         self._promote_package_version(target_package_info)
 
+    def _resolve_version_id(self) -> str:
+        """
+        If a SubscriberPackageVersionId has not been specified we need to
+        auto-resolve a version_id from the tag on our current commit
+
+        @returns: (str) the SubscriberPackageVersionId from the current repo commit
+        """
+        self.logger.info(
+            "No version_id specified. Automatically resolving to latest available Beta version."
+        )
+        tag_name = self.project_config.get_latest_tag(beta=True)
+        repo = self.project_config._get_repo()
```

Comment:
This may need some adjustment when @davidmreed's dependency branch is merged.

Reply:
@davisagli - I've gone ahead and made a 1pt work item to address this and placed it in the "2GP release support" epic.